docs/wiki/EL/el-specs.md

```diff
@@ -0,0 +1,1335 @@
+# Execution Layer Specifications
+
+> **Where is it specified?**
+>
+> - [Yellow Paper paris version 705168a – 2024-03-04](https://ethereum.github.io/yellowpaper/paper.pdf) (note: This is outdated does not take into account post merge updates)
```

Comment:
Yellowpaper is getting updated but less regularly. Unless you are pointing to a specific commit in some context, the reference doesn't have to be specific 
```suggestion
> - [Yellow Paper](https://ethereum.github.io/yellowpaper/paper.pdf) (note: Yellow paper gets updated less regularly and might be outdated)
```